src/test/java/educationalproject/programmingstuff/tests_integrations/UserControllerTest.java

```diff
@@ -53,21 +72,32 @@ public void prepareTestData() {
         User userDos = User.builder()
                 .name("John")
                 .surname("NotSmith")
-                .orders(testOrders)
                 .build();
 
         User userTres = User.builder()
                 .name("Ivan")
                 .surname("Kuznets")
-                .orders(testOrders)
                 .build();
 
         itemRepository.saveAndFlush(item);
         orderRepository.saveAndFlush(order);
         userRepository.saveAndFlush(userUno);
         userRepository.saveAndFlush(userDos);
         userRepository.saveAndFlush(userTres);
```

Comment:
I suggest to add only one string instead of these five:
userRepository.saveAll(List.of(userUno, userDos, userTres));

Hibernate should save users and all child entities